### PR TITLE
feat(replay): Add note about potential Angular performance impact

### PR DESCRIPTION
**Pull request by billyvg**
image

### CODE REVIEW

File: src/platforms/javascript/common/session-replay/index.mdx [hunk 1]
```diff
@@ -31,6 +31,10 @@ By default, it'll mask all fields, so no personally identifiable information (PI
 
 <PlatformContent includePath="session-replay/pre-requisites" />
 
+<Alert level="warning">
+  We’re investigating possible performance degradation in some Angular applications. We recommend testing your application before releasing. Please reach out via [this GitHub ticket](https://github.com/getsentry/sentry-javascript/issues/6946) if you have issues.
```
Comment (Lms24) [inline src/platforms/javascript/common/session-replay/index.mdx:35]: Just to make this a little more obvious, wdyt? 

```suggestion
  We’re investigating possible performance degradation in some **Angular applications**. We recommend testing your application before releasing. Please reach out via [this GitHub ticket](https://github.com/getsentry/sentry-javascript/issues/6946) if you have issues.
```

**Review by Lms24 — COMMENTED** [review]
Hmm, not sure if we actually need this, given that we've only received two reports so far? 
But I might be missing something/not be up to date, so feel free to merge it if y'all discussed this already.

**Comment by billyvg** [comment]
@Lms24 We talked about this briefly in a meeting, this was a potential solution in order to be extra cautious. It's hard for us to determine actual impact since we're relying on 2 layers of feedback.

**Review by Lms24 — APPROVED** [review]
Fair enough, thanks for explaining

**Review by jas-kas — CHANGES_REQUESTED** [review]
@billyvg Can we only show this on the Session Replay instructions for Angular - https://docs.sentry.io/platforms/javascript/guides/angular/session-replay/

Right now, it shows up on every single instructions for each platform (Ember, Gatsby, etc.) which I think is unnecessary and seems alarming. 

Also, the bolding and hyperlinking seems to be broken in the note : (

Screen Shot 2023-02-09 at 3 21 04 PM

File: src/platforms/javascript/common/session-replay/index.mdx [hunk 2]
```diff
@@ -31,6 +31,10 @@ By default, it'll mask all fields, so no personally identifiable information (PI
 
 <PlatformContent includePath="session-replay/pre-requisites" />
 
+<Alert level="warning">
+  We’re investigating possible performance degradation in some **Angular applications**. We recommend testing your application before releasing. Please reach out via [this GitHub ticket](https://github.com/getsentry/sentry-javascript/issues/6946) if you have issues.
+</Alert>
+
```
Comment (lizokm) [inline src/platforms/javascript/common/session-replay/index.mdx:37]:
```suggestion
<Alert level="warning">

 We’re investigating possible performance degradation in some **Angular applications**. We recommend testing your application before releasing. Please reach out via <a class href=“https://github.com/getsentry/sentry-javascript/issues/6946”>this GitHub ticket</a> if you have issues.
 
</Alert>

```
Links haven't been working in alerts/notes for some reason. ^^ they do seems to work if we write them this way.

**Review by lizokm — APPROVED** [review]
Updated the link, hopefully it's fix that and the bolding issue... but definitely agree with @jas-kas that we should only have this show up in Angular.

File: src/platforms/javascript/common/session-replay/index.mdx (hunk 2 above)
Comment (billyvg) [reply src/platforms/javascript/common/session-replay/index.mdx]: Ah looks like markdown is not supported inside of react components, thanks!

**Review by jas-kas — APPROVED** [review]

**Pull request merged**